src/App.tsx

```diff
@@ -1,31 +1,88 @@
+/* eslint-disable no-console */
 import React from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api/api';
 
 interface State {
   selectedUserId: number;
+  todos: Todo[];
+  isLoading: boolean;
 }
 
 class App extends React.Component<{}, State> {
   state: State = {
     selectedUserId: 0,
+    todos: [],
+    isLoading: false,
+  };
+
+  componentDidMount() {
+    this.loadTodos();
+  }
+
+  loadTodos = async () => {
+    this.setState({ isLoading: true });
+
+    try {
+      const todos = await getTodos();
+
+      this.setState({
+        todos,
+        isLoading: false,
+      });
+    } catch (error) {
+      console.warn(error);
```

Comment:
but common user won't see it( Try to do it as an error message, like render some info based on error condition